p4-fusion/branch_set.cc

```diff
@@ -7,13 +7,15 @@
 #include "branch_set.h"
 #include "minitrace.h"
 #include <map>
+#include <utility>
 
-static const std::string EMPTY_STRING = "";
+static const std::string EMPTY_STRING;
```

Comment:
Suggestion 1: Remove this and replace occurrences with "".

Suggestion 2: Delete the `INVALID_BRANCH_PATH` below, that's unused.